colibre/registration.py

```diff
@@ -901,3 +927,4 @@ def register_stellar_mass_scatter(self, catalogue, stellar_mass_scatter_amplitud
 register_star_Mg_and_O_to_Fe(self, catalogue, aperture_sizes_30_50_100_kpc)
 register_gas_fraction(self, catalogue)
 register_stellar_mass_scatter(self, catalogue, stellar_mass_scatter_amplitude)
+register_star_magnitudes(self, catalogue, aperture_sizes)
```

Comment:
```suggestion
register_star_magnitudes(self, catalogue, aperture_sizes_30_50_100_kpc)
```

Reply:
Yes, good point. 30kpc was the eagle choice. I should have changed it.

Can you make the change here? I am off laptop today.

Reply:
Sure, done